src/fork_gc.c

```diff
@@ -968,7 +1074,9 @@ static int periodicCb(RedisModuleCtx *ctx, void *privdata) {
     }
 
     gc->execState = FGC_STATE_APPLYING;
-    FGC_parentHandleFromChild(gc, &ret_val);
+    if (FGC_parentHandleFromChild(gc) == REDISMODULE_ERR) {
+      gcrv = 0;
```

Comment:
I think that this will cause the GC to stop, I do not think we want that..

Reply:
We might want to throttle it somehow though.. especially if there's a bug...